File: app/src/main/java/com/globallogic/weathapp/data/model/WeatherMedia.java
package com.globallogic.weathapp.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import org.apache.commons.lang3.builder.ToStringBuilder;

//@RealmClass
public class WeatherMedia {


    @SerializedName("icon")
    @Expose
    private String icon;
    @SerializedName("code")
    @Expose
    private Integer code;
    @SerializedName("description")
    @Expose
    private String description;

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    public Integer getCode() {
        return code;
    }

    public void setCode(Integer code) {
        this.code = code;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this).append("icon", icon).append("code", code).append("description", description).toString();
    }
}

File: app/src/main/java/com/globallogic/weathapp/data/model/WeatherMedia.kt
package com.globallogic.weathapp.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
//import io.realm.RealmObject
//import io.realm.annotations.RealmClass

//@RealmClass
open class WeatherMedia /*: RealmObject()*/ {

    @SerializedName("icon")
    @Expose
    var icon: String? = null

    @SerializedName("code")
    @Expose
    var code: String? = null

    @SerializedName("description")
    @Expose
    var description: String? = null
}